modules/fluid_properties/Makefile

```diff
@@ -18,9 +18,26 @@ ADDITIONAL_CPPFLAGS += -Wall -Wextra
 include $(FRAMEWORK_DIR)/build.mk
 include $(FRAMEWORK_DIR)/moose.mk
 
+################################## SALINE #######################################
+# Include link to Saline property database if the directory exists
+ifneq ("$(wildcard saline_moose/Makefile)", "")
+   #MOOSE_UNITY=false
+   APPLICATION_DIR    := $(shell pwd)/saline_moose
+   APPLICATION_NAME   := saline_moose
+   BUILD_EXEC         := no
+   ADDITIONAL_CPPFLAGS += -DSALINEAPP_H
+   ADDITIONAL_INCLUDES := -I$(APPLICATION_DIR)/include/base
+   ADDITIONAL_LIBS    += -L$(APPLICATION_DIR)/lib
+   DEP_APPS           := $(shell $(FRAMEWORK_DIR)/scripts/find_dep_apps.py $(APPLICATION_NAME))
+   include            $(FRAMEWORK_DIR)/app.mk
+   #MOOSE_UNITY=true
+endif
+
```

Comment:
This won't work if you are not directly compiling the `fluid_properties` app. Put this stuff inside `fluid_properties.mk` instead.

Reply:
Where is the `fluid_properties.mk` file located?

Reply:
Nevermind... that's what I'm adding.

Reply:
Fixed